agenda/network/src/main/java/com/pronoidsoftware/agenda/network/KtorRemoteAgendaDataSource.kt

```diff
@@ -135,17 +136,24 @@ class KtorRemoteAgendaDataSource @Inject constructor(
             body = MultiPartFormDataContent(
                 formData {
                     append(EVENT_CREATE_REQUEST, Json.encodeToString(event.toCreateEventRequest()))
-                    event.photos.forEachIndexed { index, photo ->
-                        val photoName = "photo$index"
-                        append(
-                            photoName,
-                            URL(photo.url).readBytes(),
-                            Headers.build {
-                                append(HttpHeaders.ContentType, "image/png")
-                                append(HttpHeaders.ContentDisposition, "filename=$photoName.png")
-                            },
-                        )
-                    }
+                    event.photos
+                        .filterIsInstance<Photo.Local>()
+                        .map { it.compressedPhotoUri }
+                        .forEachIndexed { index, compressedPhotoUri ->
+                            val photoName = "photo$index"
+                            append(
+                                photoName,
+                                URL(compressedPhotoUri).readBytes(),
+                                // TODO: get bytes of created compressed file
+                                Headers.build {
+                                    append(HttpHeaders.ContentType, "image/png")
```

Comment:
Did you make sure only PNGs can be attached? Or have you tried this with JPGs and it worked? (not sure how strict the server is with this metadata)